WooCommerce/Classes/ViewRelated/Orders/NewNoteViewController.swift

```diff
@@ -218,6 +227,20 @@ extension NewNoteViewController: UITableViewDelegate {
     }
 }
 
+// MARK: - Error Notice
+//
+private extension NewNoteViewController {
+    func displayErrorNotice() {
+        let title = NSLocalizedString("Unable to add note to order #\(viewModel.order.orderID)", comment: "Content of error presented when Add Note Action Failed")
```

Comment:
Perhaps add `It reads: ....` to the translator's comment. See my other comment above ☝️ 

:-)